src/locales/en.json

```diff
@@ -33,18 +29,16 @@
   "importation": {
     "cancel": "Cancel",
     "complete_failure": "Import failed.",
-    "complete_success":
-      "Congratulations! %{smart_count} contact was imported into your Cozy. |||| Congratulations! %{smart_count} contacts were imported into your Cozy.",
+    "complete_success": "Congratulations!",
+    "creation": "a contact has been created |||| %{smart_count} contacts have been created",
+    "update": "a contact has been updated |||| %{smart_count} contacts have been updated",
```

Comment:
The rest of the changes in this file is just formatting (I have prettier on my code editor).

The real changes are the three lines above.